src/foremast/s3/s3deploy.py

```diff
@@ -79,25 +80,32 @@ def _path_formatter(self, suffix):
             str: formatted path
 
         """
-        path_format = "{}/{}/{}"
+        if suffix.lower() == "flat":
```

Comment:
I'm wondering if this makes it more legible:
```
if flat:
  path = [self.bucket, self.s3path]
else:
  path = [self.bucket, self.s3path, suffix]

path_format = '/'.join(path)
```

Reply:
yeah, I like that, will update